samples/sensor/accel_trig/sample.yaml

```diff
@@ -35,3 +35,31 @@ tests:
       - SNIPPET=rtt-tracing;rtt-console
     platform_allow:
       - apard32690/max32690/m4
+  sample.sensor.accel_trig.shield-tap:
+    harness: console
+    tags: sensors
+    harness_config:
+      type: one_line
+      regex:
+        - "^\\s*[0-9A-Za-z_,+-.]*@[0-9A-Fa-f]* \\[m\/s\\^2\\]:    \
+          \\(\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*\\)$"
+    extra_args:
+      - SHIELD=x_nucleo_iks01a3
+      - EXTRA_DTC_OVERLAY_FILE=x_nucleo_iks01a3.overlay
+      - CONFIG_SAMPLE_TAP_DETECTION=y
+    depends_on:
+      - arduino_i2c
+    integration_platforms:
+      - nrf52dk/nrf52832
+  sample.sensor.accel_trig.generic-tap:
+    harness: console
+    filter: dt_compat_enabled("nxp,fxos8700") or dt_compat_enabled("invensense,icm42605") or
```

Comment:
The testcase should still specify a single `integration_platform` to make sure it is run in PR CI (in addition to these filters).

Reply:
I have added it